devito/function_manager.py

```diff
@@ -73,6 +83,47 @@ def generate_function_body(self, function_descriptor):
         statements.append(cgen.Statement("return 0"))
```

Comment:
This and in general Function manager has been improved in yask branch that we are working on now